src/PointwiseFunctions/AnalyticData/GrMhd/OrszagTangVortex.hpp

```diff
@@ -28,7 +28,7 @@ namespace AnalyticData {
  * The relativistic version of the Orszag-Tang vortex is a
  * 2-dimensional test case for relativistic MHD systems (see, e.g.,
  * [arXiv:1101.3573](https://arxiv.org/abs/1101.3573)
```

Comment:
If the arxiv links are now in the `cite` popup I guess they can be removed here :)